src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -977,6 +993,25 @@ function IOURequestStepScan({
                                 receiptImageTopPosition={receiptImageTopPosition}
                             />
                         )}
+                        {!isMobile() && isLastAppLoginLoaded && !hasInstalledApp && (
```

Comment:
With these many changes and checks, I think it makes sense to make it a resuable component.